resources/views/users/edit-add.blade.php

```diff
@@ -60,7 +71,7 @@
                                 <input type="password" class="form-control" id="password" name="password" value="" autocomplete="new-password">
                             </div>
 
-                            @can('editRoles', $dataTypeContent)
+                            @if(auth()->user()->hasPermission('edit_users'))
```

Comment:
You should be able to still use `@can()`.  Just pass `auth()->user()` instead of `$dataTypeContent`